docs/technical.rst

```diff
@@ -162,6 +173,21 @@ compiler ``tsc`` command line tool.
 .. note::
    ``TODO``: This section has to be expanded.
 
+Node.js Version
+---------------
+
+The following table gives an overview on supported Node versions for distribution:
+
+====================== ================= ===============================
+Node Version           Status            Latest Status Change
+====================== ================= ===============================
+Node.js 4              Dropped           2018-10-01
+Node.js 6              Supported         2018-10-01
+Node.js 8              Supported         2018-11-01
+Node.js 10             In the works      2018-12-01
```

Comment:
I think its safe to call Node.js 10 supported here as all existing code should run on it.

Reply:
We had some problems with Node.js 10 lately - mainly due to dependencies not being up-to-date to play well with Node.js 10, e.g. https://github.com/ethereumjs/ethereumjs-client/issues/53 - so I would like to keep this on this "in the works" state a bit until we have got a bit more assurance that most code/dependencies are in line.